packages/react-dom-bindings/src/events/DOMPluginEventSystem.js

```diff
@@ -57,10 +57,7 @@ import {
 import {createEventListenerWrapperWithPriority} from './ReactDOMEventListener';
 import {
   removeEventListener,
-  addEventCaptureListener,
-  addEventBubbleListener,
-  addEventBubbleListenerWithPassiveFlag,
-  addEventCaptureListenerWithPassiveFlag,
+  addEventListener,
```

Comment:
With this change, all event listeners now go through addEventListener, which might end up binding unnecessary listeners.